src/components/organisms/PageFooter/PageFooter.tsx

```diff
@@ -18,15 +18,26 @@ const StyledFooter = styled(Footer)`
 `;
 
 const PageFooter = () => {
+  const [appVersion, setAppVersion] = useState('');
+  const [footerText, setFooterText] = useState('');
   const fileMap = useAppSelector(state => state.main.fileMap);
   const rootEntry = fileMap[ROOT_FILE_ENTRY];
 
   // not counting the root
   const nrOfFiles = Object.keys(fileMap).length - 1;
 
-  const footerText = `Monokle ${packageJson.version} - kubeshop.io 2021${
-    rootEntry && rootEntry.children ? ` - ${rootEntry.filePath} - ${nrOfFiles} files` : ''
-  }`;
+  ipcRenderer.send('app-version');
+  ipcRenderer.once('app-version', (_, {version}) => {
+    setAppVersion(version);
+  });
+
+  useEffect(() => {
+    setFooterText(
+      `Monokle ${appVersion} - kubeshop.io 2021${
+        rootEntry && rootEntry.children ? ` - ${rootEntry.filePath} - ${nrOfFiles} files` : ''
```

Comment:
could we show in the footer if Monokle is downloading an update - and/or when a download has finished so the user knows they can install it?

Reply:
When there is an available update I show a notification.
<img width="295" alt="Screen Shot 2021-08-24 at 13 54 48" src="https://user-images.githubusercontent.com/2957624/130604795-50628068-f742-4367-aac4-b11eea5ac162.png">

And if the update is installing I show it inside the settings drawer.
<img width="400" alt="Screen Shot 2021-08-24 at 13 54 57" src="https://user-images.githubusercontent.com/2957624/130604871-44595da2-59f6-4a2f-ac2c-85bb17e11f67.png">

And When the update is ready for install. The user will see this button.
<img width="401" alt="Screen Shot 2021-08-24 at 13 55 07" src="https://user-images.githubusercontent.com/2957624/130604912-84b0ba3e-dc83-47dd-8086-e673a1f9def4.png">


Do you still think we need to show in the footer. I don't want to show a lot of notifications to users.